### DISCUSSION TITLE
Homeassistant crash / flat line - raw uart rx-fifo full

### DISCUSSION THREAD

**Discussion by ronaldt80** [OpenCCU/OpenCCU#1340] [🇺🇸 Q&A] [answered]
Hi Jens, all

Any idea what causes the below errors?

![1024-2219](https://user-images.githubusercontent.com/53364332/125162804-28bacf80-e18a-11eb-9706-dd9d83794c67.png)

These errors always precede a crash of HA; subsequently my home assistant flat lines, sometimes once per 2 days, sometimes multiple times per day. I take it the errors are related to the RF module I use for homematic? Is this a memory shortage of some sort?

I run an Odroid N2+ with multiple addons; perhaps just asking too much and need to spin off to an RPiB3+ for RaspberryMatic?

thanks!
Ronald

**Comment by jens-maus** [comment]
Can you please convert this to a proper issue ticket here on GitHub? Then I could forward this to the appropriate developer responsible for the `generic_raw_uart` kernel driver. But please make sure not to just post a screenshot from your mobile, but please copy&paste the full error log in text-form as an addition to the ticket.

**Reply by ronaldt80** [reply]
Done, hope I filled it correctly.

**Comment by ronaldt80** [comment, marked as answer]
This was indeed a memory availability issue - once I shut down some other heavy user addons the issue did not reoccur
